src/lib/Translation/Extractor/JavaScriptFileVisitor.php

```diff
@@ -85,12 +87,14 @@ public function visitFile(SplFileInfo $file, MessageCatalogue $catalogue)
         }
 
         $ast->traverse(function ($node) use ($catalogue, $file) {
-            if ($this->isMethodCall($node, self::TRANSLATOR_OBJECT, self::TRANSLATOR_METHOD)) {
+            if ($this->isMethodCall($node, self::TRANSLATOR_OBJECT, self::TRANSLATOR_TRANS_METHOD) || $this->isMethodCall($node, self::TRANSLATOR_OBJECT, self::TRANSLATOR_TRANS_CHOICE_METHOD)) {
                 $arguments = $node->getArguments();
-
                 $id = $this->extractId($file, $arguments);
                 if ($id !== null) {
-                    $message = new Message($id, $this->extractDomain($file, $arguments) ?? $this->defaultDomain);
+                    $callee = $node->getCallee();
+                    $property = $callee->getProperty();
+
+                    $message = new Message($id, $this->extractDomain($file, $arguments, $property->getName()) ?? $this->defaultDomain);
```

Comment:
CS, also line too long here:
```suggestion
                    $message = new Message(
                        $id,
                        $this->extractDomain($file, $arguments, $property->getName()) ?? $this->defaultDomain
                    );
```